recipe/meta.yaml

```
@@ -30,7 +30,7 @@ requirements:
     - dill >=0.3.6  # [py==311]
     - dill >=0.3.7  # [py>=312]
     - platformdirs >=2.2.0
-    - astroid >=3.2.4,<3.3.0
+    - astroid >=3.3.8,<=3.4.0
```

```suggestion
    - astroid >=3.3.8,<3.4.0
```
upstream allows 3.4.0dev but not higher.